e2e/playwright/.eslintrc.json

```diff
@@ -0,0 +1,18 @@
+{
+    "extends": [
+        "../../webapp/.eslintrc.json"
+    ],
+    "overrides": [
+        {
+            "files": ["**/*.ts"],
+            "plugins": ["import"],
+            "parser": "@typescript-eslint/parser",
+            "rules": {
+                "import/no-unresolved": "off",
```

Comment:
Does this cause other issues with linting? e.g. something that has to do with the type that was imported. This may be caught by the typescript compiler instead, though CI is not running checks with `tsc` in the `e2e` folder

Reply:
tsc/lint are done after cloning webapp repo in setup-environment. It happened on CI and locally